### DISCUSSION TITLE
Managing frontend assets

### DISCUSSION THREAD

**Discussion by Jovenspier** [adonisjs/core#1812] [Help] [answered]
Hello !
I recently switched from django to adonisjs because version 5 is just great.
I searched in the version 5 doc but I can't find where to deal with the frontend assets, can I use the same "Managing frontend assets" tutorial in version 4?
Thank you already!

**Comment by dawidow** [comment, marked as answer]
Hey @Jovenspier, feel free to use Laravel-mix adapter for AdonisJS. You can find it [here](https://github.com/wahyubucil/adonis-mix-asset).

**Reply by Jovenspier** [reply]
Thank you very much :)

**Reply by dawidow** [reply]
If you'll have problems with invoke that package you should remove **adonix-mix-asset** from node_modules catalog and package.json file and re-install it with command `npm i --save-dev adonis-mix-asset@next`. You can read more about this problem [here](https://github.com/wahyubucil/adonis-mix-asset/issues/1).

**Reply by Jovenspier** [reply]
Ok, i will if i have this issue. thanks again

**Reply by dawidow** [reply]
You have to install `laravel-mix@next` too. :) thats all.

**Reply by Jovenspier** [reply]
Adonisjs is my first Nodejs framework and I love it, I think it's the best thing that can happen in the world of Nodejs
